lib/std/build/RunStep.zig

```diff
@@ -37,6 +37,9 @@ stdin_behavior: std.ChildProcess.StdIo = .Inherit,
 
 expected_exit_code: u8 = 0,
 
+/// Print the command before running it
+print: bool,
```

Comment:
print implies to me `std.debug.print` and we are logging to stderr. Is there at any documented convention why/when to do this?
Argumentation below.